pkg/identity/cache/allocator.go

```diff
@@ -186,9 +186,9 @@ func (m *CachingIdentityAllocator) InitIdentityAllocator(client clientset.Interf
 	maxID := idpool.ID(identity.MaximumAllocationIdentity)
 
 	log.WithFields(map[string]interface{}{
-		"min":        minID,
-		"max":        maxID,
-		"cluster-id": option.Config.ClusterID,
+		"min":              minID,
+		"max":              maxID,
+		logfields.ClusteId: option.Config.ClusterID,
```

Comment:
We already have an ID field that we can reuse here, no need to define a new one.

Reply:
Thanks for review.
You means already have an ID field  , Is that is the 'Identity'? 
But this new field is means 'option.Config.ClusterID', maybe a cluster-id is more match for that.

Reply:
I thought we had a generic ID field, but apparently not. My only concern now is the misspelling of ClusterID:

```suggestion
		logfields.ClusterID: option.Config.ClusterID,
```

Reply:
Thanks for the suggestion.
Had done it.